src/server/schema/typeDefs.js

```diff
@@ -0,0 +1,53 @@
+const { gql } = require('apollo-server-express');
```

Comment:
This honestly looks pretty solid to me!

Honestly, a lot of the Facebook patterns I'm used to aren't as applicable with this Apollo Server, as we were using Hack for our backend server. The fact that you can so tightly couple the schema and the resolvers here is something that just wasn't possible, or at least something that we chose not to do.

I started looking into this and found this blog post: https://www.apollographql.com/blog/backend/schema-design/modularizing-your-graphql-schema-code/

Based off the blog post, they do advocate to have both the schema and the resolver for a particular query in the same file. That may be a little too much given the complexity of the project so far, but worth thinking about!

Reply:
Seeing this example (this discussion between the two) helps me better understand ways I can provide feedback and resources when doing a collaborating review. Thanks.